src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -126,7 +139,7 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
   }
 
   /**
-   * Indicates whether the checkbox is selected.
+   * Indicates whether to select the checkbox.
```

Comment:
I'd recommend keeping these descriptions the way they are - the verbiage has already been vetted and approved by our docs folks. It's also consistent with our other setters.

Reply:
Sure thing; can't remember why I decided to change that anyway. Fixed in last commit!